server/controllers/contest.js

```diff
@@ -0,0 +1,95 @@
+const asyncHandler = require("express-async-handler");
+const Contest = require('../models/Contest')
+
+// handler for creating a new contest object
+exports.createContest = asyncHandler(async (req, res, next) => {
+    const { title, description, user, prizeAmount, deadline } = req.body;
+
+    const contest = await Contest.create({ title, description, prizeAmount, deadline });
+
+    if (contest) {
+        res.status(201).json({
+            success: {
+                contest: {
```

Comment:
I think it would be better if we send the entire contest object as a response? :)

Reply:
noted.